pytest_mh/utils/tc.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from .. import MultihostHost, MultihostUtility
+from ..ssh import SSHLog, SSHProcessError
+
+__all__ = ["LinuxTrafficControl"]
+
+
+class LinuxTrafficControl(MultihostUtility):
+    """
+    Perform traffic control operations on remote host.
+
+    All changes are automatically reverted when a test is finished.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+        self.__restore = set()  # type: set[str]
+
+        # Let's find out the interface that connects the systems
+        self.__interface: str = ""
```

Comment:
I'd make this public, and use it as a default interface for tc command, not the only one.